books/kestrel/c/syntax/external-preprocessing.lisp

```diff
@@ -108,11 +108,8 @@
     '"gcc")
```

Comment:
The documentation string for `preprocessor` mentions gcc as default, and then again gcc under "other" reasonable values. Probably the second one should be cpp.

(This is not related to this PR, but something I noticed.)